borg/platform_linux.pyx

```diff
@@ -141,3 +140,47 @@ def acl_set(path, item, numeric_owner=False):
                 acl_set_file(p, ACL_TYPE_DEFAULT, default_acl)
         finally:
             acl_free(default_acl)
+
+cdef _sync_file_range(fd, offset, length, flags):
+    if sync_file_range(fd.fileno(), offset, length, flags) != 0:
+        raise OSError(errno, os.strerror(errno))
```

Comment:
maybe remove this wrapper? it just adds the raise and the .fileno - and if you did not have the wrapper, you could reuse the fileno for fadvise.